tests/utils.go

```diff
@@ -1268,11 +1268,24 @@ func RenderPod(name string, cmd []string, args []string) *k8sv1.Pod {
 // CreateExecutorPodWithPVC creates a Pod with the passed in PVC mounted under /pvc. You can then use the executor utilities to
 // run commands against the PVC through this Pod.
 func CreateExecutorPodWithPVC(podName string, pvc *k8sv1.PersistentVolumeClaim) *k8sv1.Pod {
-	return RunPod(newExecutorPodWithPVC(podName, pvc))
-}
+	var err error
+
+	pod := libstorage.RenderPodWithPVC(podName, []string{"/bin/bash", "-c", "touch /tmp/startup; while true; do echo hello; sleep 2; done"}, nil, pvc)
+	pod.Spec.Containers[0].ReadinessProbe = &k8sv1.Probe{
+		ProbeHandler: k8sv1.ProbeHandler{
+			Exec: &k8sv1.ExecAction{
+				Command: []string{"/bin/cat", "/tmp/startup"},
+			},
+		},
+	}
+	pod = RunPod(pod)
 
-func newExecutorPodWithPVC(podName string, pvc *k8sv1.PersistentVolumeClaim) *k8sv1.Pod {
-	return libstorage.RenderPodWithPVC(podName, []string{"/bin/bash", "-c", "while true; do echo hello; sleep 2;done"}, nil, pvc)
+	Eventually(func() bool {
+		pod, err = ThisPod(pod)()
+		Expect(err).ToNot(HaveOccurred())
+		return PodReady(pod) == k8sv1.ConditionTrue
+	}).Should(BeTrue())
```

Comment:
Is this intentional or should we pass timeout&interval? IIRC it defaults to some low values

Reply:
Oh oops. I guess that's where the CI is pointing to as a failure. I was wondering what has a 5 second timeout :)

Reply:
yup https://github.com/kubevirt/kubevirt/blob/2c6a11087bb1a5ce455c5880e45c2ed9e6af1b56/tests/testsuite/fixture.go#L133-L134

Reply:
Fixed!